static.config.js

```diff
@@ -109,9 +109,19 @@ export default {
         sharedData: { sidebarContent: sharedSidebarContent }
       },
       {
-        path: "/guides", // guides shares the 404 template because its children have their own docs-template getting used and there is no parent page to render, removing this will cause you build issues
-        template: "src/pages/404",
-        getData: () => ({ docs: trueDocs }),
+        // The "/guides" URL used to be a page, but is no longer. Because it is linked to from other documentation,
+        // do a redirect to the first guide.
+        path: "/guides",
+        template: "src/pages/redirect",
+        getData: () => {
+          const landerBasePathTrimmedOfSlashes = landerBasePath
+            .replace(/^\//, "")
```

Comment:
This solution won't work correctly in development, since it will make the redirect always use the base path prefixed url. I wrote a little helper that makes use of `useBasepath` from `react-static`. Check out `src/helpers/path-helpers`. It's meant to be used within the templates rather than in the static config.

Reply:
@boygirl Updated to use `createPath` -- thanks for the pointer!